yarn.lock

```diff
@@ -5945,6 +5986,18 @@ modern-normalize@^1.1.0:
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/modern-normalize/-/modern-normalize-1.1.0.tgz#da8e80140d9221426bd4f725c6e11283d34f90b7"
 
+moment-timezone@^0.5.26:
+  version "0.5.33"
+  resolved "https://registry.yarnpkg.com/moment-timezone/-/moment-timezone-0.5.33.tgz#b252fd6bb57f341c9b59a5ab61a8e51a73bbd22c"
+  integrity sha512-PTc2vcT8K9J5/9rDEPe5czSIKgLoGsH8UNpA4qZTVw0Vd/Uz19geE9abbIOQKaAQFcnQ3v5YEXrbSc5BpshH+w==
+  dependencies:
+    moment ">= 2.9.0"
+
+"moment@>= 2.9.0", moment@^2.24.0:
```

Comment:
I'm not sure why moment is here, as it's not in the package.json dependencies. Ideally we'd like to stay away from moment, as we use day.js for everything, and moment is a very bloated dependency.

Reply:
I think this was added by the: ews-javascript-api module (For the DateTime.Parse function I guess)?

Reply:
I am open if someone want to contribute to 'ews-javascript-api' and replace moment (+ timezone) with something more lightweight like dayjs ore date-fns (or suggest the best alternative)